docs/core/building/distribution-packaging.md

```diff
@@ -0,0 +1,69 @@
+---
+title: .NET Core distribution packaging
+description: .NET Core distribution packaging
+keywords: .NET, .NET Core, source, build
+author: beleroy
+ms.author: mairaw
+ms.date: 03/29/2017
+ms.topic: article
+ms.prod: .net-core
+ms.devlang: dotnet
+ms.assetid: 71b9d722-c5a8-4271-9ce1-d87e7ae2494d
+---
+
+.NET Core distribution packaging
+================================
+
+As .NET Core becomes available on more and more platforms, thanks to the efforts of dedicated contributors, it is useful to provide guidelines on how to package, name, and version it, in order to ensure a consistency of experience no matter where you choose to run .NET.
+This document provides such guidelines.
+
+What .NET Core is made of
+-------------------------
+
+.NET Core is made of three major parts that need to be packaged:
+
+1. **The host** (also known as the "muxer") has two distinct roles: activate a runtime to launch an application, and activate an SDK to dispatch commands to it. The host is a native executable (`dotnet.exe`) and its supporting policy libraries (installed in `host/fxr`). It's built from the code in the [`dotnet/core-setup`](https://github.com/dotnet/core-setup/) repository. There is typically only one host on a given machine, although that is not a strict requirement.
+2. **The framework** is made of a runtime and supporting managed libraries. A framework may be installed as part of an application, or as a shared framework in a central location that can be re-used by multiple applications. There may be any number of shared frameworks installed on a given machine. Shared frameworks live under `shared/Microsoft.NETCore.App/<version>`. The host will roll forward across patch versions, so if an application targets `Microsoft.NETCore.App` 1.0.0, and only 1.0.4 is present, it will be launched against 1.0.4.
+3. **The SDK** (also sometimes referred to as "the tooling") is a set of managed tools that can be used to write and build .NET Core libraries and applications. This includes the CLI, MS Build and associated build tasks and targets, NuGet, new project templates, etc. It is possible to have multiple SDKs on a machine (in order, for example, to allow for building projects that explicitly require an older version), but the recommendation is to use the latest available tools whenever possible.
```

Comment:
also sometimes referred to as -> also known as 

MS Build -> MSBuild

It is possible -> It's possible

use the latest available tools whenever possible: wouldn't the support come into play here? LTS and Current?